stablehlo/dialect/Version.h

```diff
@@ -75,6 +75,14 @@ mlir::Diagnostic& operator<<(mlir::Diagnostic& diag, const Version& version);
 llvm::raw_ostream& operator<<(llvm::raw_ostream& os, const Version& version);
 
 }  // namespace vhlo
+
+namespace stablehlo {
+
+/// Return the current version for StableHLO APIs.
+/// Increments on all C++/C/Python API changes
```

Comment:
As just discussed, we are planning to limit this to just changes to APIs under `integrations/` for now because this is the original use case. Can expand in the future.